File: src/gameEntities/modifiers/AttackUpgrade1.java
package gameEntities.modifiers;

import dynamicBody.character.Stats;
import worldModel.utilities.Pair;
import worldModel.utilities.enums.Entities;

public class AttackUpgrade1 extends ModifiersImpl{

	public AttackUpgrade1(Pair<Integer, Integer> coord) {
		super(coord, Stats.ATTACK, 10, "Ebony Bow", Entities.ATTACKUPGRADE1);
	}
}

File: Pirazzoli_Federico/GameEntity/AttackUpgrade1.cs
using System;

using design.utilities;
using design.enums;


namespace gameEntities.modifiers
{
    public class AttackUpgrade1 : ModifiersImpl
    {
        public AttackUpgrade1(Pair<int, int> coord) 
            : base(coord, Stats.ATTACK, 500, "Attack Upgrade", Entities.ATTACKUPGRADE)
        {

        }

    }
}
